apps/okreads-e2e/src/specs/reading-list.spec.ts

```diff
@@ -33,15 +33,15 @@ describe('When: I use the undo feature in the reading list feature', () => {
 
     // Add Book to List
     const items = await $$('[data-testing="book-item"]').first();
-    await items.getWebElement().findElement(By.tagName('button')).click();
+    await items.$$('[data-testing="add-book"]').first().click();
 
     // Open list
     const readingListToggle = await $('[data-testing="toggle-reading-list"]');
     await readingListToggle.click();
     const readingListCountBefore = await $$('[data-testing="reading-list-content"]').first().all(By.css('.reading-list-item')).length;
 
     // Remove Book
-    const removeBookButton = await $$('[data-testing="reading-list-content"]').first().getWebElement().findElement(By.tagName('button'));
+    const removeBookButton = await $$('[data-testing="reading-list-content"]').first().$$('[data-testing="remove-book"]').first();
```

Comment:
Are you using the right selector here? `$$` is the element array finder. Do you really expect there to be more than one `reading-list-content` element?

Reply:
You could just reduce this to `await $$('[data-testing="remove-book"]').first()` really.

Reply:
Thanks for the clarification. Made appropriate updates.